assets/kube-state-metrics/deployment.yaml

```diff
@@ -63,7 +63,7 @@ spec:
         - --port=8081
         - --telemetry-host=127.0.0.1
         - --telemetry-port=8082
-        image: quay.io/coreos/kube-state-metrics:v1.8.0
+        image: quay.io/coreos/kube-state-metrics:v1.9.0
```

Comment:
This is fine, we don't want 1.9.0 yet, as it still has a potential bug, but it will be overriden by CVO anyways, so "this is fine". :)